Sources/MapboxMaps/Foundation/MapView+Attribution.swift

```diff
@@ -0,0 +1,65 @@
+@_implementationOnly import MapboxCommon_Private
+
+@available(iOSApplicationExtension, unavailable)
+extension MapView: AttributionDialogManagerDelegate {
+    func viewControllerForPresenting(_ attributionDialogManager: AttributionDialogManager) -> UIViewController {
+        // Consider delegating out to a delegate
+        return parentViewController!
```

Comment:
force unwrapping seems like it could lead to errors. What about adding guards here?

Reply:
A log seems like a good idea here too.